helpers/collection/collection.go

```diff
@@ -64,22 +64,22 @@ func FormatQuery(ctx web.Context, opts Options) error {
 		return errors.NewError(http.StatusInternalServerError, err.Error())
 	}
 
-	if err = opts.Filter.ApplyPager(&opts, &response); err != nil {
+	if err = opts.Former.ApplyPager(&opts); err != nil {
 		ctx.Logger().Error(err.Error())
 		return errors.NewError(http.StatusBadRequest, err.Error())
 	}
 
-	if err = opts.Filter.ApplySorter(&opts, &response); err != nil {
+	if err = opts.Former.ApplySorter(&opts); err != nil {
 		ctx.Logger().Error(err.Error())
 		return errors.NewError(http.StatusBadRequest, err.Error())
 	}
 
 	return format(ctx, &opts, response)
 }
 
-// FormatSimple collection as response-answer (Collection)
-func FormatSimple(ctx web.Context, opts Options) error {
-	if opts.Filter == nil || opts.Fetcher == nil {
+// ResponseSimple collection as response-answer (Collection)
+func ResponseSimple(ctx web.Context, opts Options) error {
```

Comment:
Like I said, we don't have cases for this method.
Thats why, I do not see any sense in this method.